package.json

```diff
@@ -85,8 +86,12 @@
     "mime": "^1.6.0",
     "moment": "^2.29.4",
     "moment-timezone": "^0.5.34",
+    "node-cache": "^5.1.2",
```

Comment:
This project is not maintained from 2019 and written with coffeescript

Reply:
any suggestions? I love that it has expiration - that is the main purpose I use it. Moreover our SessionServiceMiddleware uses it. Looking at it's code I have no idea why it should be updated - straightforward algorithmic logic :)

Reply:
Ok, it has just 1 deps, ok )

Reply:
exactly ))